drivers/sensors/sht4x_uorb.c

```diff
@@ -875,7 +875,6 @@ int sht4x_register(FAR struct i2c_master_s *i2c, int devno, uint8_t addr)
 
   priv->hum.sensor_lower.ops = &g_sensor_ops;
   priv->hum.sensor_lower.type = SENSOR_TYPE_RELATIVE_HUMIDITY;
-  priv->hum.sensor_lower.uncalibrated = false;
```

Comment:
It would be good to update the recent documentation on UORB to mention the removal of the `uncalibrated` member.

Reply:
https://github.com/apache/nuttx/pull/15619 @linguini1  Done